vagrant/provisioning/confluent-platform.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: localhost
```

Comment:
@kikomanaskov333 and add `confluent-platform` to the arkcase-ce.yml and arkcase-dev.yml playbooks